src/components/common/PageAbout.vue

```diff
@@ -2,8 +2,15 @@
   <TmPage data-title="About" hide-header>
     <div class="card">
       <h1>About</h1>
-      <p>🎉 <router-link to="careers">We're Hiring!</router-link></p>
-      <h2 class="c5">What is Lunie?</h2>
+      <p>
+        🎉
+        <router-link to="careers">
+          We're Hiring!
+        </router-link>
+      </p>
+      <h2 class="c5">
```

Comment:
Maybe c5 could be renamed for clarity?

Reply:
@jbibla

Reply:
yeah this is bad, but i think the CMS and website will handle this. there is a lot of bad class names here.